src/style.css

```diff
@@ -1,3 +1,96 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
+* {
```

Comment:
Don't use `*` selector as it impacts performance. It's better to set styles only for elements that require them.